en-US/mozorg/about/governance/policies/community-hotline.lang

```diff
@@ -0,0 +1,25 @@
+## NOTE: Demo page: https://www-dev.allizom.org/about/governance/policies/participation/
+## URL: https://www-dev.allizom.org/%LOCALE%/about/governance/policies/participation/
+
+
+# HTML page title
+;Community Participation Guidelines - How to Report
+Community Participation Guidelines - How to Report
+
+
+# Main title
+;Community Participation Guidelines Hotline
+Community Participation Guidelines Hotline
+
+
+;The heart of Mozilla is people. We put people first and do our best to recognize, appreciate and respect the diversity of our global contributors. The Mozilla Project welcomes contributions from everyone who shares our goals and wants to contribute in a healthy and constructive manner within our community.
+The heart of Mozilla is people. We put people first and do our best to recognize, appreciate and respect the diversity of our global contributors. The Mozilla Project welcomes contributions from everyone who shares our goals and wants to contribute in a healthy and constructive manner within our community.
+
+
+;To report violations of the Community Participation Guidelines <strong>in Mozilla’s communities</strong>, please click the “Report” button below. For more information on how to take and give a report please read “<a href="%(howto)s">How to Report</a>”.
```

Comment:
`...take and give a report please read “<a href="%(howto)s".` Need to add a comma before `please`